.github/PRIVACY.md

```diff
@@ -0,0 +1,51 @@
+# Kindly Security and GDPR Policy
+
+At Kindly, the security of our codebases and the privacy of our community are of paramount importance. This document outlines our policies on security and adherence to the General Data Protection Regulation (GDPR) principles, details how to report security vulnerabilities, and explains our approach to ensuring data privacy.
+
+## Reporting a Security Vulnerability
+
+### Note
+
+If you believe you have identified a security vulnerability within our project, we urge you to report it responsibly. **Please do not open GitHub issues for security concerns.** Follow the guidelines provided below for reporting.
+
+### Guidelines for Responsible Disclosure
+
+We value the contribution of the security community and welcome responsible disclosure of any vulnerabilities that might affect the integrity of our project and the safety of its users. Before reporting, we kindly ask that you:
+
+- Verify that your findings affect the latest version of our project.
+- Avoid privacy violations, destruction of data, and interruption or degradation of our service during your research.
+- Provide us with a reasonable amount of time to address your report before making any information public.
+
+### How to Report
+
+For reporting security vulnerabilities, please email us at [hello@enbloc.uk](hello@enbloc.uk). 
+
+Upon receipt of your report, we will:
+
+- Confirm the report, assigning it a unique identifier.
+- Assess the vulnerability, its impact, and possible mitigations.
+- Keep you informed of our progress during the remediation process.
+
+## GDPR Compliance
+
+Kindly is committed to protecting the data privacy of our users in accordance with GDPR requirements. Our practices include:
+
+- Ensuring lawful, fair, and transparent data processing.
+- Limiting data processing to specified, legitimate purposes.
+- Minimising the amount of data collected.
+- Ensuring data accuracy and the possibility of erasure.
+- Keeping data secure.
+
+### Data Subject Rights
+
+Under GDPR, individuals have the right to access, correct, delete, and restrict the processing of their personal data. To exercise these rights, please contact us at [hello@enbloc.uk](hello@enbloc.uk).
+
+
+## Amendments to This Policy
+
+Kindly reserves the right to amend this policy at any time. The most current version will always be posted on our website.
```

Comment:
Same here - "the core development team managing Kindly reserves the right..."

Reply:
I agree, I have changed it so that it is referring to enBloc